tests/mocha/procedure_map_test.js

```diff
@@ -8,18 +8,18 @@ import {sharedTestSetup, sharedTestTeardown} from './test_helpers/setup_teardown
 
 goog.declareModuleId('Blockly.test.procedureMap');
 
-suite.skip('Procedure Map', function() {
+suite('Procedure Map', function() {
   setup(function() {
     sharedTestSetup.call(this);
     this.workspace = new Blockly.Workspace();
-    this.procedureMap = this.workspace.getProcedureMap();
+    // this.procedureMap = this.workspace.getProcedureMap();
   });
 
   teardown(function() {
     sharedTestTeardown.call(this);
   });
 
-  suite('triggering block updates', function() {
+  suite.skip('triggering block updates', function() {
```

Comment:
just checking this is supposed to be skipped now? can you add a TODO comment saying when to unskip it?

Reply:
Yesh! The skip just got moved from the top-level down to here. Can add a comment o7